vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/internal/CVCInputField.kt

```diff
@@ -105,8 +141,42 @@ internal class CVCInputField(context: Context): BaseInputField(context) {
 
     override fun setupAutofill() {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            setAutofillHints(View.AUTOFILL_HINT_CREDIT_CARD_SECURITY_CODE )
+            setAutofillHints(View.AUTOFILL_HINT_CREDIT_CARD_SECURITY_CODE)
+        }
+    }
+
+    private fun refreshIcon() {
+        when (previewIconVisibility) {
+            ALWAYS -> setIcon(getIcon())
+            HAS_CONTENT -> setIcon(if (text.isNullOrEmpty()) null else getIcon())
+            IF_BRAND_DETECTED -> setIcon(if (cardType != CardType.UNKNOWN) getIcon() else null)
```

Comment:
Rewrite please HAS_CONTENT and IF_BRAND_DETECTED cases. This style is not comfortable for reading and could be a problem in a future.
Initialize if-else as VAL value and than place it to setIcon(_), or format code into separate lines, or inherit style from other places with such cases.

Reply:
@Endorf This is basic implementation it will change in the next PR's